src/Watchy.cpp

```diff
@@ -138,6 +140,7 @@ void Watchy::handleButtonPress() {
       }
       showMenu(menuIndex, true);
     } else if (guiState == WATCHFACE_STATE) {
+      button3();
```

Comment:
```suggestion
      handleButton4Press();
```

And this one :slightly_smiling_face: 

![image](https://user-images.githubusercontent.com/820984/168224875-aeddeb23-5e49-4543-9b0f-8058a01004da.png)